specifyweb/frontend/js_src/lib/components/Merging/Status.tsx

```diff
@@ -99,10 +100,7 @@ export function Status({
           {state.status === 'MERGING' && (
             <>
               <Progress max={state.total} value={state.current} />
-              <RemainingLoadingTime
-                current={state.current}
-                total={state.total}
-              />
+              <p>{percentage}%</p>
```

Comment:
hmm, I don't like this change.
the video Carlos sent looked like a bug - if so, it needs to be fixed rather than the feature removed

Reply:
Changing the remaining time by a percentage is what as been decided during the Sp meeting with the all team

Reply:
The thought process behind the change is that the time remaining is often sporadic by nature when it comes to merging. 
 The current and total in the requests have a tendency to "jump around" a lot. More specifically, whenever the merging process needs to merge records recursively, the 'total' will need to be updated to reflect the number of added columns, which may cause the `time remaining` function to fluctuate. 

The thought process was that using a percentage "handles" the visual turbulence better from a user standpoint, since it is not _heavily_ changing the progress UI based on a drastic change in the current and total. 
In other words, a percentage is hopefully smoother. 
(Not my own personal thoughts here, just what I picked up as the reasoning behind the change)

> the video Carlos sent looked like a bug - if so, it needs to be fixed rather than the feature removed

While it may be tricky to have the time remaining function handle results like how it might get them form agent merging, I agree this is a bug and should probably be looked into/fixed

Reply:
makes sense.

this is why the progress bars in any software with good UX are mostly fake.
they just gradually move across, and get slower the closer they reach to the end, and get faster the more actual progress is completed - until finally when everything is actually complete, instead of disappearing right away (which would break the illusion), the scroll bar reaches the end in a span of 1-2 seconds - giving immense satisfaction to the user

macOS update progress bar might be the best example of this
though, windows have also gotten much better with their progress bars since windows xp days